tsl/src/continuous_aggs/materialize.c

```diff
@@ -73,6 +73,77 @@ static InternalTimeRange materialization_invalidation_log_get_range(int32 materi
 static void materialization_invalidation_log_delete_or_cut(int32 cagg_id,
 														   InternalTimeRange invalidation_range,
 														   int64 completed_threshold);
+
```

Comment:
This code was moved from `refresh.c`, but otherwise remains the same.

Reply:
Code not reviewed in that case. Please tell me if I should.